src/include/OpenImageIO/fmath.h

```diff
@@ -897,7 +897,7 @@ inline T safe_pow (T x, T y) {
     // less than FLOAT_MAX (but still bigger than anyone is likely to
     // need) so that subsequent additions or multiplications are less
     // likely to overflow and end up with an Inf right afterwards.
-    const T big = T(1e30);
+    const T big = std::numeric_limits<T>::max();
```

Comment:
Ah, right. Do you think those "subsequent overflows" are likely to happen? There are only a few particular ops where we try to clamp Inf overflows. We certainly don't do it for ordinary multiplication or addition. I guess returning FLOAT_MAX means we have no wiggle room at all, but I don't honestly know if it matters.